lib/slack-ruby-bot-server/api/presenters/team_presenter.rb

```diff
@@ -10,7 +10,7 @@ module TeamPresenter
         property :team_id, type: String, desc: 'Slack team ID.'
         property :name, type: String, desc: 'Team name.'
         property :domain, type: String, desc: 'Team domain.'
-        property :active, type: Boolean, desc: 'Team is active.'
+        property :active, type: "Boolean", desc: 'Team is active.'
```

Comment:
Yup. It doesn't. Does keeping it as a string make a big difference for the Mongo impl.?